docker/volumes/kong.yml

```diff
@@ -76,10 +132,31 @@ services:
           - /storage/v1/
     plugins:
       - name: cors
-consumers:
-  - username: anon
-    keyauth_credentials:
-      - key: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYyNzIwODU0MCwiZXhwIjoxOTc0MzYzNzQwfQ.zcaQfHd3VA7XgJmdGfmV86OLVJT9s2MTmSy-e69BpUY
-  - username: service_role
-    keyauth_credentials:
-      - key: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlIjoic2VydmljZV9yb2xlIiwiaWF0IjoxNjI3MjA4NTQwLCJleHAiOjE5NzQzNjM3NDB9.pkT3PNpO4DtO45Ac5HK_TKCx8sGLgNtV__pr_ZrRSAU
+      - name: key-auth
```

Comment:
this might have been for publicly available files and buckets (images etc.)? @inian should be able to verify

Reply:
yeah, storage doesn't need key-auth from kong. it manages its own authentication

Reply:
is it still useful to keep a layer of protection in Kong? Block requests at the edge?

Reply:
we could add in a open route/service for any public storage routes? Up to you - happy to revert if it's completely unnecessary